internal/route/routes.go

```diff
@@ -76,7 +78,7 @@ func HandleTodos(w http.ResponseWriter, r *http.Request) {
 		w.WriteHeader(http.StatusNotFound)
 		return
 	}
-	list, err := model.GetTodosListByID(listID)
+	list, err := model.TodoListByID(listID)
```

Comment:
Just using our new method. The comment on this method tells us it fetches. But we don't need to guess the prefix "Get" to find it in the IDE. We can just start typing "Todo..." and see all the exported methods that can help us use the model package.